lib/npg_pipeline/function/seq_alignment.pm

```diff
@@ -426,7 +426,7 @@ sub _alignment_command { ## no critic (Subroutines::ProhibitExcessComplexity)
     $p4_param_vals->{reference_dict} = $self->_ref($dp, q(picard)) . q(.dict);
     $p4_param_vals->{reference_genome_fasta} = $self->_ref($dp, q(fasta));
     if($self->p4s2_aligner_intfile) { $p4_param_vals->{align_intfile_opt} = 1; }
-    $p4_param_vals->{markdup_method} = $self->markdup_method($dp);
+    $p4_param_vals->{markdup_method} = ($self->is_paired_read ? $self->markdup_method($dp) : q[samtools]);
```

Comment:
Wasn't expecting this dependent on target alignment - though that may be appropriate.

We were considering something like:
```
if (not  $self->is_paired_read and not $self->markdup_method($dp) ) {
  # override markdup method for single read runs as we experience occasional hangs using default biobambam
  $p4_param_vals->{markdup_method} = q[samtools];
}
```

Reply:
You are right - this will not work since it doesn't change phix or other subsets when there is no target alignment.

But I think it should be just:
if (not  $self->is_paired_read) {
  # override default markdup method for single read runs as we experience  
  # occasional hangs using default (biobambam)
  $p4_param_vals->{markdup_method} = q[samtools];
  $self->info(q[Overriding markdup method for single-end, always use samtools]);
}

In other words, use markdup_method:samtools for single-end even if product_release.yml says otherwise. Do you agree?

Reply:
I think that is ok: my use of `and not $self->markdup_method($dp)` was to _try_ to make it conditional on the use of the default.